js/app.js

```diff
@@ -131,18 +131,38 @@ class WebKitBug extends Bug {
 }
 
 class MozillaBug extends Bug {
+
+  static get BUG_PREFIX() {
+    return 'https://bugzilla.mozilla.org/rest/bug/';
+  }
+
   constructor(url) {
     super(url);
-    this.needsCors = true;
+    this.needsCors = false;
   }
 
-  findStatus(doc) {
-    const statusEl = doc.querySelector('#static_bug_status');
-    if (statusEl) {
-      return statusEl.textContent.replace('\n', '')
-                                 .replace(/ +(?=)/g, ' ').trim();
+  fetchPage(url=this.url) {
+    let match = url.match(/\?id=(.*)$/);
+    if (!match) {
+      return Promise.reject('Could not find Mozilla bug id.');
```

Comment:
Minor: This reads like the bug can't be found in the DB and not in the URL.  'Could not find Mozilla bug id in query string'